data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+			label distress
+			`	Just before you are overcome by the inexplicable sleepiness, you reach over and hit the big "Distress" button. You can only hope that everything works alright.`
+			choice
+				`	(Continue.)`
+			branch poor
+				"net worth" < 30000000
+			`	You find yourself awake in a room, which is definitely not your own. It is lit by a luxurious glass chandelier hanging from a lofty groined ceiling. Looking around, there isn't much furniture, apart from the bed, a few scattered chairs, a row of cabinets to the left, and a large display taking nearly the whole wall in front. The floor is made of white and red marble tiles.`
+				goto doctor
+			label poor
+			`	You find yourself awake in a room, which is definitely not your own. It is lit solely by a few windows that reveal a bustling spaceport nearby. There isn't much furniture apart from the bed and a cabinet.`
+			label doctor
+			`	As you muse and wonder where you are, someone looking like a doctor or perhaps a nurse enters the room.`
+			`	"You are lucky to be still living", she says, "You barely survived an encounter with a recently discovered disease tentatively known as X-3625.".`
+			choice
+				`	"Can you tell me more about this disease?"`
+				`	"Is there anything I should do about it?"`
+					goto do
+				`	"Yep, just like me. Missing every danger through sheer luck."`
+					goto luck
+			`	"The disease appears to be caused by a certain bacteria of unknown origin. It seems to be quite picky about its surroundings. It thrives best in low gravity environments, gradually spreading over the host's body. When exposed to an environment with higher gravity, such as during landing, the bacteria rapidly swell in size, effectively blocking most of the blood flow and causing a sensation of fatigue and sleepiness."`
+			choice
+				`	"Is there anything I should do about it?"`
+					goto do
+			label luck
+			branch poorluck
+				"net worth" < 10000000
+			`	"I wouldn't count on luck so much. A risk-averse way of life and a well-written will is worth more than any lucky star out there. But you are a spaceship captain, I guess you wouldn't understand."`
+			choice
+				`	"Is there anything I should do about this disease?"`
+					goto do
+			label poorluck
+			`	"I wouldn't count on luck so much. A modest way of life and an industrious mentality is worth more than any lucky star out there. But you are a spaceship captain, I guess you wouldn't understand."`
+			choice
+				`	"Is there anything I should do about this disease?"`
+			label do
+			`	"We already injected an ample amount of the proper medicine. It will not cure you instantly, but it contains certain compounds which inhibit the pathogen's reproduction cycle, causing it to die out soon. Naturally, the life-saving operation and medicine are not free. The cost totals to 100,000 credits."`
```

Comment:
Endless Sky doesn't believe in Universal Healthcare.

Reply:
Even if it does, have you paid the respective taxes first? :smiley: